.github/sync-repo-settings.yaml

```diff
@@ -19,21 +19,24 @@ branchProtectionRules:
       - 'Kokoro - Test: Java GraalVM Native Image'
       - 'Kokoro - Test: Java 17 GraalVM Native Image'
       - javadoc
-  - pattern: 1.106.5-sp
+  - pattern: V3-experimental
```

Comment:
diff is a little confusing here, can you just edit the required checks for this branch and leave the others as is?